### ISSUE TITLE
Shoulder instability

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10993]
**Preferred term label:**
Shoulder instability

**Synonyms**

**Definition (free text, please give PubMed ID)**
Subjective or objective feeling of clunking/clicking in the shoulder with movement, shoulder 'giving way' or sensation of shoulder dislocating

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Shoulder dislocation, Labrum tear, shoulder, Suprascapular neuropathy

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001224
- label: Shoulder instability
- parents: [Abnormal shoulder physiology (HP:0033129)](https://hpo.jax.org/app/browse/term/HP:0033129)
- pmid: 
- def: Subjective or objective feeling of clunking/clicking in the shoulder with movement, shoulder 'giving way', or sensation of shoulder dislocating.
- comment: Shoulder instability may be observed in conditions such as shoulder dislocation, labrum tear, and suprascapular neuropathy.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10993
- synonyms:
